pkg/http/server/server_test.go

```diff
@@ -32,7 +32,10 @@ func family(name string, timestamps ...int64) *clientmodel.MetricFamily {
 func storeWithData(data map[string][]*clientmodel.MetricFamily) store.Store {
 	s := memstore.New()
 	for k, v := range data {
-		if err := s.WriteMetrics(context.TODO(), k, v); err != nil {
+		if err := s.WriteMetrics(context.TODO(), &store.PartitionedMetrics{
```

Comment:
nit, the `if err := ...; err != nil` pattern is nice for one-liners, but for multi-line statements becomes quite hard to read. I would suggest breaking this into multiple lines

Reply:
will do